### PR TITLE
[oPHST 2018] Various additions and changes

### PR DESCRIPTION
**Pull request by LeNitrous**
### Description
- Schedule and Ruleset changes and additions
- Staff additions
- Participant additions
- Clarification in some sections
- Minor typo fixes

### Status

- [ ] WIP
- [x] Pending Review
- [ ] Finished

### Additional Notes 
None

### CODE REVIEW

File: wiki/Tournaments/oPHST/2018/en.md [hunk 1]
```diff
@@ -43,13 +44,16 @@ This also serves as the third edition of the annual **osu!mania Philippines Summ
...
 | Qualifiers (4K)      | 2018-04-28/2018-04-29 |
+| Play-offs  (4K)      | 2018-05-05/2018-05-06 |
 | Qualifiers (7K)      | 2018-05-05/2018-05-06 |
 | Group Stage (4K)     | 2018-05-12/2018-05-13 |
 | Group Stage (7K)     | 2018-05-19/2018-05-20 |
 | Quarterfinals        | 2018-05-26/2018-05-27 |
 | Semifinals           | 2018-06-02/2018-06-03 |
 | Finals Week 1        | 2018-06-09/2018-06-10 |
-| Finals Week 1        | 2018-06-06/2018-06-17 |
+| Finals Week 2        | 2018-06-06/2018-06-17 |
```
Comment (TPGPL) [inline wiki/Tournaments/oPHST/2018/en.md]: aren't those Finals timestamps broken?

**Comment by TPGPL** [comment]
update the fork before making any changes

**Comment by LeNitrous** [comment]
Should be ready, updated master and branch with main repo.

File: wiki/Tournaments/oPHST/2018/en.md [hunk 2]
```diff
@@ -97,6 +97,21 @@ An initial list of participants may be found on [this spreadsheet](https://docs.
 
 Mappools will be revealed one week in advance of each stage.
 
+### Qualifiers
+
+#### osu!standard
+
+[**Download the Mappack here! (22.6 MB)**](https://drive.google.com/open?id=1c9jrOMN_UqeSPeaifCcMGxaDB9M-YGrc)
```
Comment (TPGPL) [inline wiki/Tournaments/oPHST/2018/en.md]: mappack

**Pull request merged**